.github/workflows/charm-release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release Charm
+
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        description: "Name of artifact to download before building. Must contain the file artifact.tar.gz."
+        default: ''
+        required: false
+        type: string
+    secrets:
+      CHARMHUB_TOKEN:
```

Comment:
Is this really needed, since `release-charm` job required it. What is difference here?

Reply:
Good catch. I guess you meant that file `_charm-release.yaml` already it is declaring `CHARMHUB_TOKEN` and there is no reason for declaring here again, right?  Observability team used like this an I didn't  contest.

I've removed from here and it still works. I guess setting `secrets: inherit` does a good job.